content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Veneers"
+linkTitle: "Veneers"
+weight: 5
+date: 2022-06-30
+---
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions.  FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/).  Unfortunately, FBCs can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`, a simpler transitional step to generating an FBC.
+
+## Design
+
+There are two components to every `veneer`:
+1. An arbitrary API
+2. An executable which processes #1 and produces a valid FBC.
+
+## Benefits
+
+- A set of consistent expectations from including as standard functionality with `opm`
+- A good set of use-cases for simplifying the ability of operator authors to manage FBCs, including contributing them to an index
+- A transitive benefit from consistency of practice as authors move from operator to operator
+
+The veneers which accompany `opm` are:
+- [`basic veneer`](#basic-veneer), which provides simplification of FBC concepts; and
+- [`semver veneer`](#semver-veneer), which provides the capability of sophisticated channel generation
```

Comment:
In this case, I understand that we do not need to config replace/skips/skipRange. Am I right

The veneer will create the upgrade graph based on the server of the bundles.
On top of that, it also manages the channel creation based on the server which makes things eaiser for authors does not need to configure them. 

So, should not batter the small description say that this veener create the upgrade graph and channels using the bundle versions follow the semver convention and link here the convention. Wdyt?